src/h256.rs

```diff
@@ -1,112 +1,45 @@
+use numext_fixed_hash;
 use core::cmp::Ordering;
 
-/// Represent 256 bits
-#[derive(Eq, PartialEq, Debug, Default, Hash, Clone, Copy)]
-pub struct H256([u8; 32]);
+pub type H256 = numext_fixed_hash::H256;
```

Comment:
This shall be a private type declaration, or at least crate-level public. We don't want to confuse the users of this library with yet another H256 definition.